scripts/hallucinate_fragalysis.py

```diff
@@ -117,23 +133,49 @@ def write_fragalysis_output(in_dir, out_dir, sort_key="POSIT_prob"):
         prot = oechem.OEGraphMol()
         du.GetProtein(prot)
         lig = load_openeye_sdf(f"{tmp_in_dir}/docked.sdf")
+
+        ## Set ligand title
         lig.SetTitle(f"{compound_id}_{best_str}")
 
         ## First save apo
         save_openeye_pdb(prot, f"{tmp_out_dir}/{best_str}_apo.pdb")
 
-        ## Combine protein and ligand and save
+        ## Combine protein and ligand
         oechem.OEAddMols(prot, lig)
+
+        # ## Get number of residues per chain
+        # num_res_chain_dict = {}
+        # hv = oechem.OEHierView(prot)
+        # for chain in hv.GetChains():
+        #     max_resid = 0
+        #     for frag in chain.GetFragments():
+        #         frag_max = max(
+        #             [r.GetResidueNumber() for r in frag.GetResidues()]
+        #         )
+        #         if frag_max > max_resid:
+        #             max_resid = frag_max
+        #     num_res_chain_dict[chain.GetChainID()] = max_resid
+        # ## Rename and renumber ligand residue
+        # for a in prot.GetAtoms():
+        #     r = oechem.OEAtomGetResidue(a)
+        #     if r.GetName() == "UNL":
+        #         r.SetHetAtom(True)
+        #         r.SetName("LIG")
+        #         r.SetResidueNumber(num_res_chain_dict[r.GetChainID()] + 100)
+        #     oechem.OEAtomSetResidue(a, r)
```

Comment:
Not a blocker. Is this code expected to be uncommented and used eventually? If not it should be better if we get rid of it :)